src/Util/TraversableToCollection.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Util;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+final class TraversableToCollection
+{
+    /**
+     * @param mixed $value
```

Comment:
```suggestion
     * @param Collection|\Traversable|array $value
```

Reply:
I you do this, the TypeError is reported as never thrown ; and static analysis is failing.